mod/match.php

```diff
@@ -48,17 +48,23 @@ function match_content(&$a) {
 			
 			$tpl = get_markup_template('match.tpl');
 			foreach($j->results as $jj) {
+			    $match_nurl = str_replace(array('https:','//www.'), array('http:','//'), $jj->url);
+			    $match = q("SELECT `nurl` FROM `contact` WHERE `uid` = '%d' AND nurl='%s' LIMIT 1",
+				intval(local_user()),
+				$match_nurl);
```

Comment:
> And why not simply use "normalise_link"? This would be the standard
> behaviour.

Because I searched for normalise_url so did not find normalise_link...